lms/templates/fields/field_social_link_account.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="u-field-value field <%- linkClass %>">
+    <label class="u-field-title field-label" for="u-field-link-<%- id %>-default"><%- title %></label>
```

Comment:
@AlasdairSwan It was mentioned on the ticket that we don't need to show username on connected accounts under the social media title, please check 6th point on the ticket under changes.